token/cli/src/main.rs

```diff
@@ -1354,12 +1596,55 @@ fn command_gc(
         }
     }
 
-    Ok(Some((lamports_needed, instructions)))
+    let cli_signer_info = CliSignerInfo {
+        signers: bulk_signers,
+    };
+
+    let mut result = String::from("");
+    for tx_instructions in instructions {
+        let tx_return = handle_tx(
+            &cli_signer_info,
+            config,
+            no_wait,
+            lamports_needed,
+            tx_instructions,
+        )?;
+        result += &match tx_return {
+            TransactionReturnData::CliSignature(signature) => {
+                config.output_format.formatted_string(&signature)
+            }
+            TransactionReturnData::CliSignOnlyData(sign_only_data) => {
+                config.output_format.formatted_string(&sign_only_data)
+            }
+        };
+        result += "\n";
```

Comment:
It would be better if the signature or sign-data list could be compiled into one Cli struct so that it will output one single json array, instead of multiple json maps separated by lines.
Do you mind fixing in a follow-up PR?

Reply:
yea I agree this would be better. I just wanted to keep it like it was before this change for this PR so I don't change too much. happy to do it in a follow-up pr yes